src/components/IconList/IconList.js

```diff
@@ -124,7 +124,17 @@ class IconList extends PureComponent {
       }
 
       return ReactDOM.createPortal(
-        <div className="wkp-icon-list-modal">
+        <div
+          role="button"
+          tabIndex={0}
+          className="wkp-icon-list-modal"
+          onClick={() => {
+            this.setState({
+              iconListVis: false,
+            });
+          }}
+          onKeyPress={() => {}}
+        >
```

Comment:
Core change

> but we also can't go out of this modal, unless we click on the select box. It's annoying. Can't you make the modal bg transparent and on a user interaction on it, you close the select box ?

from https://github.com/geops/trafimage-maps/pull/453#discussion_r402965394

I have added an onClick listener which closes the dropdown.

However, the modal dropdown cannot be used/closed by keys. As it is appended to the .tm-trafimage-maps element we would have to manually change the focus. With an onFocus callback function for the react-ui Button component we could do this. @oterral, what do you think?

Reply:
> With an onFocus callback function for the react-ui Button component we could do this.

@oterral, The onFocus callback function would not solve this problem. I have to react on the onClick function

Reply:
We won't support barrier free behavior, for now.